app/services/slack_client.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class SlackClient
+  class Error < StandardError
+  end
+
+  def self.create_message(message)
+    new.create_message(message)
+  end
+
+  def create_message(message)
+    response = client.post('', { text: message })
```

Comment:
```suggestion
    response = client.post('/', { text: message })
```

I think this is just a little bit clearer, wdyt?